app/helpers/xml.py

```diff
@@ -141,6 +141,14 @@ def build_mh_mets(g: rdflib.Graph, pid: str) -> str:
     for representation in representations:
```

Comment:
Interpreteer ik dit correct? Op deze manier wordt:
 - een meemoo SIP representatie gemapt op een MH SIP media;
 - een meemoo SIP file gemapt op een MH SIP representatie die gelinkt is aan bovenstaande media.

Eigenlijk wordt de structuur van een meemoo SIP representatie met onderliggende files op die manier behouden in de MH SIP.  Is dat de bedoeling? Ik dacht dat elke meemoo SIP file, ging gemapt worden naar een unieke MH SIP media met achterliggende MH SIP representatie. Zo interpreteer ik toch die afbeeldingen op hun wiki. OF is dat fout?